doc/source/indexing.rst

```diff
@@ -508,6 +508,81 @@ A list of indexers where any element is out of bounds will raise an
 
 .. _indexing.basics.partial_setting:
 
+Selecting Random Samples
+------------------------
+.. versionadded::0.16.1
+
+A random selection of rows or columns from a Series, DataFrame, or Panel with the ``.sample()`` method. The method will sample rows by default, and accepts a specific number of rows/columns to return, or a fraction of rows. 
```

Comment:
can you replace ``` ``.sample()`` ```` with `` :meth:`~DataFrame.sample` `` (of `` :meth:`Series.sample` ``), the result will be almost the same, but also be a link to the api docstring page